notebooks/en/_toctree.yml

```diff
@@ -68,4 +68,6 @@
   - local: enterprise_cookbook_overview
     title: Overview
   - local: enterprise_cookbook_argilla
-    title: Data annotation with Argilla Spaces
\ No newline at end of file
+    title: Data annotation with Argilla Spaces
+  - local: enterprise_hub_serverless_inference_api
```

Comment:
put your notebook above the argilla notebook to maintain the ordering in the overview file: https://huggingface.co/learn/cookbook/enterprise_cookbook_overview#data-annotation-with-argilla-spaces